public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -165,7 +164,7 @@ export default class InfluxDatasource extends DataSourceWithBackend<InfluxQuery,
       return super.query(filteredRequest);
     }
 
-    if (config.featureToggles.influxdbBackendMigration && this.access === 'proxy' && request.app === CoreApp.Explore) {
+    if (config.featureToggles.influxdbBackendMigration && this.access === 'proxy') {
```

Comment:
nit. We see this in multiple places. Can we move this to a common function so that we can reuse?